Classes/Domain/Model/AbstractType.php

```diff
@@ -92,7 +92,7 @@ public function findDocumentById($id)
      * @param string $content
      * @return Response
      */
-    public function request($method, $path = null, array $arguments = [], $content = null)
+    public function request($method, $path = null, array $arguments = [], $content = null, $header = null)
```

Comment:
please specify the the type of the `$header` attribute and mention it in the commet